CHANGELOG.md

```diff
@@ -5,8 +5,11 @@
         - Plugins supported: `table`
     - API:
       - Support for exits numbers and names. New member `exits` in `RouteStep`, based on `junction:ref` on ways
+      - `RouteStep` now has new parameter `classes` that can be set in the profile on each way.
```

Comment:
> This is not about road classes? This flag is user-defined, it doesn't expose our road-class hierarchy.

Ah sorry I think I have a different understanding for "road classes" than our road class hierarchy. Thanks for the explanations, all clear here.